iam/api-client/src/test/java/iam/snippets/QuickstartTests.java

```diff
@@ -108,7 +108,9 @@ public void tearDown() {
 
     String resource = "projects/-/serviceAccounts/" + serviceAccount.getEmail();
     try {
-      iamService.projects().serviceAccounts().delete(resource).execute();
+      if (resource != null) {
```

Comment:
I don't think `resource` can be ever `null` as we declare and assign this variable in line 109.